_caseStudies/rhl.html

```diff
@@ -1,4 +1,7 @@
 ---
 layout: default
 sitemap: false
+thumbnail: "/assets/images/work/rhlThumbnail.jpg"
+title: RHL.co.uk
+published: "n"
```

Comment:
Published stops it being included in the list of case studies. I want to keep them both separate. Agreed, you could use a single var for both though.